structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,250 @@
+
+This role is very similar to the Community Manager role within Quilt's own Community Team.
+
+### Collaborators
+
+The Collaborator role is given to moderators from the communities that are part of the Program, and who have joined a Program-related space in order to represent their communities. Administrators from these communities are welcome to join as Collaborators as well, if they wish.
+
+Collaborators are given access to all moderation-related areas within Program spaces, such as general moderation chat channels and case-tracking threads. Additionally, all Collaborators are granted access to spaces dedicated to organizing events between participating communities.
+
+### Individual Collaborators
+
+The Individual Collaborator role is given to moderators that have joined the Program, but do not represent any specific community, which may be for any number of reasons. We believe that Individual Collaborators should be able to make use of the Program, and can contribute to it just how regular Collaborators can.
+
+Individuals are given the same level of access that regular Collaborators are given. However, Individual Collaborators are not given voting rights --- this is something that still needs to be discussed, but for the sake of simplicity and ease of implementation, this discussion has been deferred until later.
+
+### Event Planners
+
+The Event Planner role is given to members of participating communities' community teams that focus on event creation, execution and management. They do not have access to moderation-related areas within Program spaces, and are instead given their own dedicated spaces.
+
+### Project Roles
+
+Project-specific roles may be created and granted to Program members that are contributing to (or participating in) Program-related projects. The definition of these roles will depend on the project in question, and they'll be created or granted on an as-needed basis.
+
+These roles aren't detailed in this RFC, as they don't impact how the Program functions.
+
+### Invitees
+
+Invitees are temporary users with no access to Program-related spaces, other than temporary areas that are created for whatever purpose they're visiting the respective space.
+
+## Voting process
+
+Votes may be called by Program Managers, where each of the communities participating in the Program represent the poll's voters. As the Program exists largely on Discord, a poll and an associated thread are created on the Program's Discord server, in a channel designated for this purpose.
+
+The poll should be as detailed as possible in order to help everyone to make an informed decision. Discussions related to the poll should happen in the associated thread, to try to keep everything together.
+
+Each community is assigned one single vote. Individual communities are responsible for coming up with their own process for determining their vote, but it must follow these rules:
```

Comment:
I have spoken about my feelings on votes being per-community and not per-collaborator, and I know I am in the minority, but I will put them here for the record:
- Members of smaller communities get more votes. Quilt is twice the size of the next-largest community, so we essentially get less than half the say of anyone else, despite Community Collab being managed by us. Not that I think we should get **more** weight than others relative to our size, but we certainly shouldn't get less.
- People who are members of multiple communities get multiple votes. Among other things, that means more well-known or prestigious community members could get a louder voice than those who may only be part of a single community. It may be a little dramatic to say that this goes against Quilt's value of making sure everyone has an equal voice, no matter their status, but I think it's an important point to make.
- In addition to the size of the community determining the voting power of a collaborator, their voting system (which is not standardised) could also have an influence on how much say they have.
- We have a problem with whether individual collaborators get votes, well, there's an easy solution.

Reply:
While I appreciate your concerns, we held a vote inside the collab server and the majority decided to go for one vote per community. This isn't something I would be comfortable changing now. We can always revisit it later with another poll if needs be.

![image](https://user-images.githubusercontent.com/44778521/198698705-2db556f3-b284-4492-8799-cbbf5dc64253.png)